### ISSUE TITLE
Cannot get rid of "Unable to sync SteamVR saves" popup

### ISSUE THREAD

**Issue by DasaniSamurai** [monstermac77/mixed-vr#20]
I've been getting this pop up every time I boot steamvr that's been happening for the past two weeks, I've spent days searching online for solutions and toggling steam cloud on/off along with trying to manipulate what's getting uploaded to my cloud but it just will not go away. Any ideas anyone? Link below is a pic of the pop up

This problem only persists when I have Mixed VR Manager enabled, which makes troubleshooting even more fun

https://i.gyazo.com/3fadccbe0e82c29168a8ce563e819e3e.png

**Comment by monstermac77** [comment]
I have this issue occasionally as well and it drives me nuts. I'll look into it at some point.

**Comment by DasaniSamurai** [comment]
It's nearly made me see red because if Steam simply had an option label "fuck off with all cloud messages" then it wouldn't be an issue but I guess they're pushing some sort of agenda because an option like that would be so easy to implement on their end imo, someone told me that they think it's related to an unspecified recent steamvr update? Fingers crossed

**Comment by monstermac77** [comment]
Maybe, but I think I was having it even as long as 2 years ago. But it comes in goes, so it could be update related.

**Comment by Turtana** [comment]
Same problem. After grinding my teeth for a few weeks, I made a duct tape solution. Do you guys like Python?

```
import pyautogui # needs to be installed
import time

# Edit according to Steam localization. It's the title of the sync error window
sync_error_title = "Steam-valintaikkuna"

# "Play anyway" button location
offset_x = 180
offset_y = 230

while True:
    windows = pyautogui.getAllWindows()
    vr_on = False
    error = None
    for w in windows:
        if w.title == "Windows Mixed Reality":
            vr_on = True
        if w.title == sync_error_title:
            error = w

    if vr_on and error:
        click_x = error.left + offset_x
        click_y = error.top + offset_y
        pyautogui.click(click_x, click_y)
    
    time.sleep(1)
```
The script dismisses the sync error as soon as it appears. Beware: it might affect other Steam windows while VR is active, and I have zero clue how it affects the performance. Works on my machine...

To run automatically, save it as a .pyw file in the startup folder, same place where you keep the MixedVR manager script.

**Comment by monstermac77** [comment]
That's great, @Turtana. So the sync error window pops up repeatedly (without this script), right? Like maybe once every 5 seconds? 

And yes, I love Python :) I wish this damn repo didn't have to be in Batch.

**Comment by Turtana** [comment]
Nah, just once every time you open SteamVR. When you click "play anyway", you're done until SteamVR is closed. But it gets annoying if you're already wearing headset and controllers and have to go back to the computer to dismiss the popup, and use VR daily.

The script solves the problem (for me anyway) but I'd rather see this fixed on the manager's side.

**Comment by DasaniSamurai** [comment]
Thank you @Turtana !

**Comment by Turtana** [comment]
The last big SteamVR update seems to have brought some good news and some bad news. On one hand, the sync error doesn't appear anymore. I removed my script.

On the other hand, screenshots don't work anymore :P Every screenshot results in a black rectangle now. My guess is that this is caused by my old Vive HMD, which is still connected to track the controllers, and SteamVR is getting the screenshot data from there.

Maybe worth opening another issue. Is anyone having the same problem?

**Comment by monstermac77** [comment]
Sure, feel free to open another issue, but do you think that could be because of MixedVR manager?
